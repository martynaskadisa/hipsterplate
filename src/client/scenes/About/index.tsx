import * as React from 'react';
import Helmet from 'react-helmet';
import { RouteComponentProps } from 'react-router-dom';

const About: React.SFC<RouteComponentProps<{}>> = () => {
  return (
    <div>
      <Helmet>
        <title>About</title>
      </Helmet>
      About
    </div>
  );
};

export default About;
