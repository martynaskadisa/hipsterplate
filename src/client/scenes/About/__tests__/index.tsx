import * as React from 'react';
import * as renderer from 'react-test-renderer';
import About from '../index';

describe('About scene', () => {
  it('should render correctly', () => {
    const tree = renderer.create(<About />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
