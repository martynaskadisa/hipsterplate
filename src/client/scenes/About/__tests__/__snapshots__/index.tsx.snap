// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About scene should render correctly 1`] = `
<div>
  About
</div>
`;
